### PR TITLE
US-001 layout header

### PR DESCRIPTION
**Pull request by jmarkusson**
The layout header for the application, a seperate folder container header.tsx and header.module.css in the components folder, 
Also linking to 3 seperate pages: visualization, events and about(3 empty pages)

### CODE REVIEW

File: agile-app/src/app/händelser.tsx
Comment (Aoc67310) [inline agile-app/src/app/händelser.tsx:1]: Swedish named file should be in english.

File: agile-app/src/app/about.tsx
Comment (willayy) [inline agile-app/src/app/about.tsx:1]: Is this file removed from the repo?

File: agile-app/src/app/visualisering.tsx
Comment (Aoc67310) [inline agile-app/src/app/visualisering.tsx:1]: Swedish named file should be in english.

File: agile-app/src/app/händelser.tsx
Comment (Aoc67310) [inline agile-app/src/app/händelser.tsx:1]: Swedish named file should be in english.

File: agile-app/src/app/layout.tsx [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
 };
-
+/**
+ * The root layout of the entire application
+ * @param param0 
+ * @returns 
+ */
 export default function RootLayout({
   children,
-}: Readonly<{
```
Comment (willayy) [inline agile-app/src/app/layout.tsx:14]: Don't remove read only its against type script conventions see the following.
Mark the props of the component as read-only typescript:S6759

File: agile-app/src/app/layout.tsx [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "Brottkollen.se",
+  description: "Sveriges bästa brottssajt",
 };
-
+/**
+ * The root layout of the entire application
+ * @param param0
```
Comment (willayy) [inline agile-app/src/app/layout.tsx:14]: param0 does not exist in the function

Comment (willayy) [inline agile-app/src/app/layout.tsx:14]: param0 does not exist in the function

File: agile-app/src/components/header/header.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ * @returns an HTML header element
+ */
+export default function Header() {
+  return (
+    <header className={styles.headerContainer}>
+      <h1 className={styles.headerTitle}>Brottkollen.se</h1> 
+      <nav className={styles.nav}>
+        <ul className="flex gap-x-6">
+        <li>
+            <Link href="/pages/händelser">Händelser</Link>
```
Comment (willayy) [inline agile-app/src/components/header/header.tsx:14]: file names should not be in Swedish

**Review by Aoc67310 — CHANGES_REQUESTED** [review]

File: agile-app/src/components/header/header.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+/**
+ * displays the header of the entire application, links to the different pages
+ * @returns an HTML header element
+ */
+export default function Header() {
+  return (
+    <header className={styles.headerContainer}>
+      <h1 className={styles.headerTitle}>Brottkollen.se</h1> 
+      <nav className={styles.nav}>
+        <ul className="flex gap-x-6">
```
Comment (willayy) [inline agile-app/src/components/header/header.tsx:12]: Nice solution!

**Review by willayy — CHANGES_REQUESTED** [review]

**Review by willayy — APPROVED** [review]

**Review by willayy — APPROVED** [review]

**Review by willayy — APPROVED** [review]

**Review by willayy — APPROVED** [review]

**Comment by willayy** [comment]
Since Lucas isn't available at the time and my final review includes his points as well you can merge!
Good work.

**Pull request merged**